### DISCUSSION TITLE
WebUI无法访问

### DISCUSSION THREAD

**Discussion by knight-mao** [chatchat-space/Langchain-Chatchat#1903] [Q&A]
好心大佬帮我瞄一眼，为什么启动所有服务后，web UI端无法访问呀。
只能访问    Chatchat  API  Server: http://127.0.0.1:7861，但是Chatchat WEBUI Server: http://127.0.0.1:8501显示无法连接，每次访问WEBUI就显示 307 Temporary Redirenct
![image](https://github.com/chatchat-space/Langchain-Chatchat/assets/42996469/5267a7ed-8467-4aa8-bee9-453dcebb2f8f)

全部log如下：
PS E:\OneDrive\Coding\AI\Langchain-Chatchat\Langchain-Chatchat-master> python startup.py -a
==============================Langchain-Chatchat Configuration==============================
操作系统：Windows-10-10.0.19045-SP0.
python版本：3.10.12 | packaged by Anaconda, Inc. | (main, Jul  5 2023, 19:01:18) [MSC v.1916 64 bit (AMD64)]
项目版本：v0.2.6
langchain版本：0.0.323. fastchat版本：0.2.31

当前使用的分词器：ChineseRecursiveTextSplitter
当前启动的LLM模型：['chatglm2-6b'] @ cuda
{'device': 'cuda',
 'host': '127.0.0.1',
 'infer_turbo': False,
 'model_path': 'E:\\OneDrive\\Coding\\AI\\Langchain-Chatchat\\Embeding-LLM-models\\chatglm2-6b',
 'port': 20002}
当前Embbedings模型： m3e-base @ cuda
==============================Langchain-Chatchat Configuration==============================

2023-10-28 08:43:07,085 - startup.py[line:626] - INFO: 正在启动服务：
2023-10-28 08:43:07,085 - startup.py[line:627] - INFO: 如需查看 llm_api 日志，请前往 E:\OneDrive\Coding\AI\Langchain-Chatchat\Langchain-Chatchat-master\logs
2023-10-28 08:43:09 | ERROR | stderr | [32mINFO[0m:     Started server process [[36m14212[0m]
2023-10-28 08:43:09 | ERROR | stderr | [32mINFO[0m:     Waiting for application startup.
2023-10-28 08:43:09 | ERROR | stderr | [32mINFO[0m:     Application startup complete.
2023-10-28 08:43:09 | ERROR | stderr | [32mINFO[0m:     Uvicorn running on [1mhttp://127.0.0.1:20000[0m (Press CTRL+C to quit)
2023-10-28 08:43:16 | INFO | model_worker | Register to controller
2023-10-28 08:43:16 | INFO | model_worker | Loading the model ['chatglm2-6b'] on worker d35066db ...
Loading checkpoint shards:   0%|                                                                 | 0/7 [00:00<?, ?it/s]
Loading checkpoint shards:  14%|████████▏                                                | 1/7 [00:01<00:11,  1.84s/it]
Loading checkpoint shards:  29%|████████████████▎                                        | 2/7 [00:03<00:08,  1.80s/it]
Loading checkpoint shards:  43%|████████████████████████▍                                | 3/7 [00:05<00:07,  1.80s/it]
Loading checkpoint shards:  57%|████████████████████████████████▌                        | 4/7 [00:07<00:05,  1.74s/it]
Loading checkpoint shards:  71%|████████████████████████████████████████▋                | 5/7 [00:09<00:03,  1.82s/it]
Loading checkpoint shards:  86%|████████████████████████████████████████████████▊        | 6/7 [00:10<00:01,  1.85s/it]
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 7/7 [00:12<00:00,  1.60s/it]
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 7/7 [00:12<00:00,  1.72s/it]
2023-10-28 08:43:29 | ERROR | stderr |
2023-10-28 08:43:34 | INFO | model_worker | Register to controller
INFO:     Started server process [4928]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:7861 (Press CTRL+C to quit)

==============================Langchain-Chatchat Configuration==============================
操作系统：Windows-10-10.0.19045-SP0.
python版本：3.10.12 | packaged by Anaconda, Inc. | (main, Jul  5 2023, 19:01:18) [MSC v.1916 64 bit (AMD64)]
项目版本：v0.2.6
langchain版本：0.0.323. fastchat版本：0.2.31

当前使用的分词器：ChineseRecursiveTextSplitter
当前启动的LLM模型：['chatglm2-6b'] @ cuda
{'device': 'cuda',
 'host': '127.0.0.1',
 'infer_turbo': False,
 'model_path': 'E:\\OneDrive\\Coding\\AI\\Langchain-Chatchat\\Embeding-LLM-models\\chatglm2-6b',
 'port': 20002}
当前Embbedings模型： m3e-base @ cuda

服务端运行信息：
    OpenAI API Server: http://127.0.0.1:20000/v1
    Chatchat  API  Server: http://127.0.0.1:7861
    Chatchat WEBUI Server: http://127.0.0.1:8501
==============================Langchain-Chatchat Configuration==============================

      Welcome to Streamlit!

      If you’d like to receive helpful onboarding emails, news, offers, promotions,
      and the occasional swag, please enter your email address below. Otherwise,
      leave this field blank.

      Email:  INFO:     127.0.0.1:51136 - "GET / HTTP/1.1" 307 Temporary Redirect
INFO:     127.0.0.1:51136 - "GET /docs HTTP/1.1" 200 OK
INFO:     127.0.0.1:51136 - "GET /swagger-ui.css HTTP/1.1" 200 OK
INFO:     127.0.0.1:51137 - "GET /swagger-ui-bundle.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:51137 - "GET /openapi.json HTTP/1.1" 200 OK
INFO:     127.0.0.1:51149 - "GET / HTTP/1.1" 307 Temporary Redirect
INFO:     127.0.0.1:51149 - "GET /docs HTTP/1.1" 200 OK
INFO:     127.0.0.1:51149 - "GET /openapi.json HTTP/1.1" 200 OK

另外conda list如下：
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Knight/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Knight/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Name | Version | Build | Channel
-- | -- | -- | --
accelerate | 0.24.0 | pypi_0 | pypi
aiohttp | 3.8.6 | pypi_0 | pypi
aiosignal | 1.3.1 | pypi_0 | pypi
altair | 5.1.2 | pypi_0 | pypi
antlr4-python3-runtime | 4.9.3 | pypi_0 | pypi
anyio | 3.7.1 | pypi_0 | pypi
async-timeout | 4.0.3 | pypi_0 | pypi
attrs | 23.1.0 | pypi_0 | pypi
backoff | 2.2.1 | pypi_0 | pypi
beautifulsoup4 | 4.12.2 | pypi_0 | pypi
blas | 1 | mkl | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
blinker | 1.6.3 | pypi_0 | pypi
blis | 0.7.11 | pypi_0 | pypi
brotli-python | 1.0.9 | py310hd77b12b_7 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bzip2 | 1.0.8 | he774522_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ca-certificates | 2023.08.22 | haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cachetools | 5.3.2 | pypi_0 | pypi
catalogue | 2.0.10 | pypi_0 | pypi
cchardet | 2.1.7 | pypi_0 | pypi
certifi | 2023.7.22 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cffi | 1.16.0 | pypi_0 | pypi
chardet | 5.2.0 | pypi_0 | pypi
charset-normalizer | 3.3.1 | pypi_0 | pypi
click | 8.1.7 | pypi_0 | pypi
cloudpathlib | 0.16.0 | pypi_0 | pypi
colorama | 0.4.6 | pypi_0 | pypi
coloredlogs | 15.0.1 | pypi_0 | pypi
confection | 0.1.3 | pypi_0 | pypi
contourpy | 1.1.1 | pypi_0 | pypi
cryptography | 41.0.5 | pypi_0 | pypi
cuda-cccl | 12.3.52 | 0 | nvidia
cuda-cudart | 12.1.105 | 0 | nvidia
cuda-cudart-dev | 12.1.105 | 0 | nvidia
cuda-cupti | 12.1.105 | 0 | nvidia
cuda-libraries | 12.1.0 | 0 | nvidia
cuda-libraries-dev | 12.1.0 | 0 | nvidia
cuda-nvrtc | 12.1.105 | 0 | nvidia
cuda-nvrtc-dev | 12.1.105 | 0 | nvidia
cuda-nvtx | 12.1.105 | 0 | nvidia
cuda-opencl | 12.3.52 | 0 | nvidia
cuda-opencl-dev | 12.3.52 | 0 | nvidia
cuda-profiler-api | 12.3.52 | 0 | nvidia
cuda-runtime | 12.1.0 | 0 | nvidia
cudatoolkit | 11.8.0 | hd77b12b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cycler | 0.12.1 | pypi_0 | pypi
cymem | 2.0.8 | pypi_0 | pypi
dataclasses-json | 0.6.1 | pypi_0 | pypi
ebooklib | 0.18 | pypi_0 | pypi
effdet | 0.4.1 | pypi_0 | pypi
einops | 0.7.0 | pypi_0 | pypi
emoji | 2.8.0 | pypi_0 | pypi
et-xmlfile | 1.1.0 | pypi_0 | pypi
exceptiongroup | 1.1.3 | pypi_0 | pypi
faiss-cpu | 1.7.4 | pypi_0 | pypi
fastapi | 0.104.0 | pypi_0 | pypi
filelock | 3.12.4 | pypi_0 | pypi
filetype | 1.2.0 | pypi_0 | pypi
flatbuffers | 23.5.26 | pypi_0 | pypi
fonttools | 4.43.1 | pypi_0 | pypi
freetype | 2.12.1 | ha860e81_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
frozenlist | 1.4.0 | pypi_0 | pypi
fschat | 0.2.31 | pypi_0 | pypi
fsspec | 2023.10.0 | pypi_0 | pypi
giflib | 5.2.1 | h8cc25b3_3 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
gitdb | 4.0.11 | pypi_0 | pypi
gitpython | 3.1.40 | pypi_0 | pypi
greenlet | 3.0.1 | pypi_0 | pypi
h11 | 0.14.0 | pypi_0 | pypi
httpcore | 0.18.0 | pypi_0 | pypi
httpx | 0.25.0 | pypi_0 | pypi
huggingface-hub | 0.17.3 | pypi_0 | pypi
humanfriendly | 10 | pypi_0 | pypi
idna | 3.4 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
importlib-metadata | 6.8.0 | pypi_0 | pypi
iniconfig | 2.0.0 | pypi_0 | pypi
intel-openmp | 2023.1.0 | h59b6b97_46319 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
iopath | 0.1.10 | pypi_0 | pypi
jinja2 | 3.1.2 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
joblib | 1.3.2 | pypi_0 | pypi
jpeg | 9e | h2bbff1b_1 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jsonpatch | 1.33 | pypi_0 | pypi
jsonpointer | 2.4 | pypi_0 | pypi
jsonschema | 4.19.1 | pypi_0 | pypi
jsonschema-specifications | 2023.7.1 | pypi_0 | pypi
kiwisolver | 1.4.5 | pypi_0 | pypi
langchain | 0.0.323 | pypi_0 | pypi
langchain-experimental | 0.0.34 | pypi_0 | pypi
langcodes | 3.3.0 | pypi_0 | pypi
langdetect | 1.0.9 | pypi_0 | pypi
langsmith | 0.0.52 | pypi_0 | pypi
layoutparser | 0.3.4 | pypi_0 | pypi
lerc | 3 | hd77b12b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libcublas | 12.1.0.26 | 0 | nvidia
libcublas-dev | 12.1.0.26 | 0 | nvidia
libcufft | 11.0.2.4 | 0 | nvidia
libcufft-dev | 11.0.2.4 | 0 | nvidia
libcurand | 10.3.4.52 | 0 | nvidia
libcurand-dev | 10.3.4.52 | 0 | nvidia
libcusolver | 11.4.4.55 | 0 | nvidia
libcusolver-dev | 11.4.4.55 | 0 | nvidia
libcusparse | 12.0.2.55 | 0 | nvidia
libcusparse-dev | 12.0.2.55 | 0 | nvidia
libdeflate | 1.17 | h2bbff1b_1 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libffi | 3.4.4 | hd77b12b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libjpeg-turbo | 2.0.0 | h196d8e1_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libnpp | 12.0.2.50 | 0 | nvidia
libnpp-dev | 12.0.2.50 | 0 | nvidia
libnvjitlink | 12.1.105 | 0 | nvidia
libnvjitlink-dev | 12.1.105 | 0 | nvidia
libnvjpeg | 12.1.1.14 | 0 | nvidia
libnvjpeg-dev | 12.1.1.14 | 0 | nvidia
libpng | 1.6.39 | h8cc25b3_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libtiff | 4.5.1 | hd77b12b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libuv | 1.44.2 | h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libwebp | 1.3.2 | hbc33d0d_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libwebp-base | 1.3.2 | h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
lxml | 4.9.3 | pypi_0 | pypi
lz4-c | 1.9.4 | h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
markdown | 3.5 | pypi_0 | pypi
markdown-it-py | 3.0.0 | pypi_0 | pypi
markdown2 | 2.4.10 | pypi_0 | pypi
markupsafe | 2.1.3 | pypi_0 | pypi
marshmallow | 3.20.1 | pypi_0 | pypi
matplotlib | 3.8.0 | pypi_0 | pypi
mdurl | 0.1.2 | pypi_0 | pypi
mkl | 2023.1.0 | h6b88ed4_46357 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl-service | 2.4.0 | py310h2bbff1b_1 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl_fft | 1.3.8 | py310h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl_random | 1.2.4 | py310h59b6b97_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mpmath | 1.3.0 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
msg-parser | 1.2.0 | pypi_0 | pypi
multidict | 6.0.4 | pypi_0 | pypi
murmurhash | 1.0.10 | pypi_0 | pypi
mypy-extensions | 1.0.0 | pypi_0 | pypi
networkx | 3.2 | pypi_0 | pypi
nh3 | 0.2.14 | pypi_0 | pypi
nltk | 3.8.1 | pypi_0 | pypi
numexpr | 2.8.7 | pypi_0 | pypi
numpy | 1.24.4 | pypi_0 | pypi
numpy-base | 1.26.0 | py310h65a83cf_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
olefile | 0.46 | pypi_0 | pypi
omegaconf | 2.3.0 | pypi_0 | pypi
onnx | 1.14.1 | pypi_0 | pypi
onnxruntime | 1.15.1 | pypi_0 | pypi
openai | 0.28.1 | pypi_0 | pypi
opencv-python | 4.8.1.78 | pypi_0 | pypi
openjpeg | 2.4.0 | h4fc8c34_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
openpyxl | 3.1.2 | pypi_0 | pypi
openssl | 3.0.11 | h2bbff1b_2 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
packaging | 23.2 | pypi_0 | pypi
pandas | 2.0.3 | pypi_0 | pypi
pathlib | 1.0.1 | pypi_0 | pypi
pdf2image | 1.16.3 | pypi_0 | pypi
pdfminer-six | 20221105 | pypi_0 | pypi
pdfplumber | 0.10.2 | pypi_0 | pypi
peft | 0.5.0 | pypi_0 | pypi
pillow | 9.5.0 | pypi_0 | pypi
pip | 23.3 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pluggy | 1.3.0 | pypi_0 | pypi
portalocker | 2.8.2 | pypi_0 | pypi
preshed | 3.0.9 | pypi_0 | pypi
prompt-toolkit | 3.0.39 | pypi_0 | pypi
protobuf | 4.24.4 | pypi_0 | pypi
psutil | 5.9.6 | pypi_0 | pypi
pyarrow | 13.0.0 | pypi_0 | pypi
pyclipper | 1.3.0.post5 | pypi_0 | pypi
pycocotools | 2.0.7 | pypi_0 | pypi
pycparser | 2.21 | pyhd3eb1b0_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pydantic | 1.10.13 | pypi_0 | pypi
pydeck | 0.8.1b0 | pypi_0 | pypi
pygments | 2.16.1 | pypi_0 | pypi
pymupdf | 1.22.5 | pypi_0 | pypi
pymupdfb | 1.23.5 | pypi_0 | pypi
pyopenssl | 23.2.0 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pypandoc | 1.12 | pypi_0 | pypi
pyparsing | 3.1.1 | pypi_0 | pypi
pypdfium2 | 4.22.0 | pypi_0 | pypi
pyreadline3 | 3.4.1 | pypi_0 | pypi
pysocks | 1.7.1 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytesseract | 0.3.10 | pypi_0 | pypi
pytest | 7.4.3 | pypi_0 | pypi
python | 3.10.12 | he1021f5_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python-dateutil | 2.8.2 | pypi_0 | pypi
python-decouple | 3.8 | pypi_0 | pypi
python-docx | 1.0.1 | pypi_0 | pypi
python-iso639 | 2023.6.15 | pypi_0 | pypi
python-magic | 0.4.27 | pypi_0 | pypi
python-magic-bin | 0.4.14 | pypi_0 | pypi
python-multipart | 0.0.6 | pypi_0 | pypi
python-pptx | 0.6.21 | pypi_0 | pypi
pytorch | 2.1.0 | py3.10_cuda12.1_cudnn8_0 | pytorch
pytorch-cuda | 12.1 | hde6ce7c_5 | pytorch
pytorch-mutex | 1 | cuda | pytorch
pytz | 2023.3.post1 | pypi_0 | pypi
pywin32 | 306 | pypi_0 | pypi
pyyaml | 6.0.1 | py310h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
rapidfuzz | 3.4.0 | pypi_0 | pypi
rapidocr-onnxruntime | 1.3.8 | pypi_0 | pypi
referencing | 0.30.2 | pypi_0 | pypi
regex | 2023.10.3 | pypi_0 | pypi
requests | 2.31.0 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
rich | 13.6.0 | pypi_0 | pypi
rpds-py | 0.10.6 | pypi_0 | pypi
safetensors | 0.4.0 | pypi_0 | pypi
scikit-learn | 1.3.2 | pypi_0 | pypi
scipy | 1.11.3 | pypi_0 | pypi
sentence-transformers | 2.2.2 | pypi_0 | pypi
sentencepiece | 0.1.99 | pypi_0 | pypi
setuptools | 68.0.0 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
shapely | 2.0.2 | pypi_0 | pypi
shortuuid | 1.0.11 | pypi_0 | pypi
simplejson | 3.19.2 | pypi_0 | pypi
six | 1.16.0 | pypi_0 | pypi
smart-open | 6.4.0 | pypi_0 | pypi
smmap | 5.0.1 | pypi_0 | pypi
sniffio | 1.3.0 | pypi_0 | pypi
soupsieve | 2.5 | pypi_0 | pypi
spacy | 3.7.2 | pypi_0 | pypi
spacy-legacy | 3.0.12 | pypi_0 | pypi
spacy-loggers | 1.0.5 | pypi_0 | pypi
sqlalchemy | 2.0.19 | pypi_0 | pypi
sqlite | 3.41.2 | h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
srsly | 2.4.8 | pypi_0 | pypi
starlette | 0.27.0 | pypi_0 | pypi
streamlit | 1.27.2 | pypi_0 | pypi
streamlit-aggrid | 0.3.4.post3 | pypi_0 | pypi
streamlit-antd-components | 0.2.3 | pypi_0 | pypi
streamlit-chatbox | 1.1.10 | pypi_0 | pypi
streamlit-option-menu | 0.3.6 | pypi_0 | pypi
svgwrite | 1.4.3 | pypi_0 | pypi
sympy | 1.12 | pypi_0 | pypi
tabulate | 0.9.0 | pypi_0 | pypi
tbb | 2021.8.0 | h59b6b97_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tenacity | 8.2.3 | pypi_0 | pypi
thinc | 8.2.1 | pypi_0 | pypi
threadpoolctl | 3.2.0 | pypi_0 | pypi
tiktoken | 0.5.1 | pypi_0 | pypi
timm | 0.9.8 | pypi_0 | pypi
tk | 8.6.12 | h2bbff1b_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tokenizers | 0.14.1 | pypi_0 | pypi
toml | 0.10.2 | pypi_0 | pypi
tomli | 2.0.1 | pypi_0 | pypi
toolz | 0.12.0 | pypi_0 | pypi
torch | 2.1.0 | pypi_0 | pypi
torchaudio | 2.1.0 | pypi_0 | pypi
torchvision | 0.16.0 | pypi_0 | pypi
tornado | 6.3.3 | pypi_0 | pypi
tqdm | 4.66.1 | pypi_0 | pypi
transformers | 4.34.1 | pypi_0 | pypi
transformers-stream-generator | 0.0.4 | pypi_0 | pypi
typer | 0.9.0 | pypi_0 | pypi
typing-extensions | 4.8.0 | pypi_0 | pypi
typing-inspect | 0.9.0 | pypi_0 | pypi
tzdata | 2023.3 | pypi_0 | pypi
tzlocal | 5.2 | pypi_0 | pypi
unstructured | 0.10.26 | pypi_0 | pypi
unstructured-inference | 0.7.10 | pypi_0 | pypi
unstructured-pytesseract | 0.3.12 | pypi_0 | pypi
urllib3 | 2.0.7 | pypi_0 | pypi
uvicorn | 0.23.2 | pypi_0 | pypi
validators | 0.22.0 | pypi_0 | pypi
vc | 14.2 | h21ff451_1 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
vs2015_runtime | 14.27.29016 | h5e58377_2 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wasabi | 1.1.2 | pypi_0 | pypi
watchdog | 3.0.0 | pypi_0 | pypi
wavedrom | 2.0.3.post3 | pypi_0 | pypi
wcwidth | 0.2.8 | pypi_0 | pypi
weasel | 0.3.3 | pypi_0 | pypi
websockets | 12 | pypi_0 | pypi
wheel | 0.41.2 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
win_inet_pton | 1.1.0 | py310haa95532_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xformers | 0.0.22.post7 | pypi_0 | pypi
xlrd | 2.0.1 | pypi_0 | pypi
xlsxwriter | 3.1.9 | pypi_0 | pypi
xz | 5.4.2 | h8cc25b3_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
yaml | 0.2.5 | he774522_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
yarl | 1.9.2 | pypi_0 | pypi
zipp | 3.17.0 | pypi_0 | pypi
zlib | 1.2.13 | h8cc25b3_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zstd | 1.5.5 | hd43e919_0 | https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main

</body>

</html>

**Comment by YXXXDD** [comment]
hxd 解决了吗

**Reply by YXXXDD** [reply]
我填了邮箱地址就好了

**Comment by Yida147** [comment]
我通过nginx stream代理解决了这个问题
以下是我的配置：
stream {
  server  {
        listen 9200;
        proxy_pass [ip]:8501;
      }
}

**Reply by liyiyiya** [reply]
这个要写在哪里呀

**Reply by Yida147** [reply]
这个是nginx的配置，需要先安装nginx

**Comment by CRonaldo1997** [comment]
同样问题，我把服务器防火墙关了就好了：
service firewalld stop

**Reply by liyiyiya** [reply]
这个是在哪里关的呀

**Comment by srwalpha** [comment]
我遇到同样问题，试了各种方法后才发现。。。
在启动后出现下面这段信息后


 Welcome to Streamlit!

 If you’d like to receive helpful onboarding emails, news, offers, promotions,
 and the occasional swag, please enter your email address below. Otherwise,
 leave this field blank.

  Email: 

按一下回车，只要一次就好，以后这个问题就解决了。

**Reply by liyiyiya** [reply]
我启动之后什么也没有呀，没有shu'r输入邮箱的地方mmm
![image](https://github.com/chatchat-space/Langchain-Chatchat/assets/166470931/bf5aa0d9-97f5-4c1d-8f4e-2c2e8c74760c)
![image](https://github.com/chatchat-space/Langchain-Chatchat/assets/166470931/f1a87baf-52d4-4ec0-b911-977fc4832c9c)

**Reply by YXXXDD** [reply]
那就不太清楚了
